excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/CategoryValidator.java

```diff
@@ -0,0 +1,165 @@
+package uk.gov.hmcts.ccd.definition.store.excel.validation;
+
+import com.microsoft.applicationinsights.boot.dependencies.apachecommons.lang3.StringUtils;
+import com.microsoft.applicationinsights.boot.dependencies.apachecommons.lang3.tuple.Pair;
+import com.microsoft.applicationinsights.boot.dependencies.apachecommons.lang3.tuple.Triple;
+import lombok.val;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception.InvalidImportException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.ParseContext;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionDataItem;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.ColumnName;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeEntity;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.groupingBy;
+
+@Component
+public class CategoryValidator {
+
+    private ParseContext parseContext;
+    private static final String ERROR_MESSAGE = "CategoryTab Invalid ";
+    private static final String NOT_VALID = " is not a valid ";
+    private static final String DISPLAY_ORDER_ERROR = "DisplayOrder cannot be duplicated within case type"
+        + " category and parent category in Category tab";
+
+
+    public CategoryValidator() {
+    }
+
+    public void validate(ParseContext parseContext, List<DefinitionDataItem> categoryItems) {
+        this.parseContext = parseContext;
```

Comment:
We shouldn't be setting the class-level field as this is a `@Component` - instead just pass the `parseContext` into the methods we need (which only looks to be `validateCaseType()`)